content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
@beorn7 This is the first one that's claimed to be a wrapper and has additional text calling that out in this PR.

It's actually a pile of utility functions which I'm mostly okay with, but some are clearly bad practice (app name shouldn't go in to the metric name, that's what the job label is for; and having an outcome label).

I don't think we should be directing Clojure users to this rather than the Java client.